lib/ansible/module_utils/facts/system/service_mgr.py

```diff
@@ -98,14 +98,13 @@ def collect(self, module=None, collected_facts=None):
             try:
                 proc_1_root_stat = os.stat('/proc/1/root')
                 self_root_stat = os.stat('/proc/%d/root' % os.getpid())
-                if proc_1_root_stat.st_dev != self_root_stat.st_dev :
+                if proc_1_root_stat.st_dev != self_root_stat.st_dev:
                     proc_1 = None
-                if proc_1_root_stat.st_ino != self_root_stat.st_ino :
+                if proc_1_root_stat.st_ino != self_root_stat.st_ino:
```

Comment:
we already have chroot detection, which covers more cases tha this test

Also, you can have chroot and still have a service mgr as proc 1, depends on the host configuration and how you create the chroot/jail.